docs/docs/.vuepress/theme/components/NavLinks.vue

```diff
@@ -141,6 +145,14 @@ export default {
   .repo-link {
     margin-left: 0.75rem;
   }
+
+  .docs-version {
+    .dropdown-title, .nav-link {
+      font-family: 'JetBrains Mono', monospace;
```

Comment:
This visually distinguished it well for me.

Separately, should we add the "New" text you have in the drop-down menu to the "v2" label in the nav, as well? If not not a big deal, but that might also bring attention to it for users who want to explore.

<img width="132" alt="Screen Shot 2022-01-21 at 2 03 41 PM" src="https://user-images.githubusercontent.com/242668/150606227-d1f4eb4e-2585-4c99-aa50-8f7ec9b07a20.png">

Reply:
I don't think the ROI is worth it. We're going to need to upgrade the whole layout and instead of incrementally hacking on my hacks.

The theme shows it's age and would really like to use a Tailwind powered theme instead of hacking styles within components.

Reply:
Totally fine